src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -133,6 +149,25 @@ const ContributionsPage = () => {
   );
 
   const { control, reset, resetField, setValue } = useForm({ mode: 'all' });
+  const { control: contributionTextControl, handleSubmit } =
+    useForm<contributionTextSchema>({
+      resolver: zodResolver(schema),
+      mode: 'all',
+    });
+  const isAdmin = isUserAdmin(me);
+  const onSubmit: SubmitHandler<contributionTextSchema> = async data => {
+    try {
+      await updateCircle({
+        circle_id: selectedCircle.id,
+        team_sel_text: data.team_sel_text,
+      });
+
+      refetchContributions();
```

Comment:
i'm not sure this is needed.. ?

Reply:
the default value of contribution help text depends on the fetched data

Reply:
hmm, but it seems wasteful to re-run the whole query when you've only changed one piece of text, and you already know what the changed value is. better to store the changed value with setState and then show it directly when it's available, i.e.
```
[updatedTeamSelText, setUpdatedTeamSelText] = setState();

<Text p>{updatedTeamSelText || data?.circles_by_pk?.team_sel_text || 'What have you been working on?'</Text>
```

Reply:
done